com.adsbynimbus.nimbus/Runtime/Scripts/NimbusManager.cs

```diff
@@ -445,6 +448,37 @@ public void SetCoppa(bool isCoppa) {
 			_nimbusPlatformAPI.SetCoppaFlag(isCoppa);
 		}
 		
+		/// <summary>
+		///     Sets the Age and Gender of the User
+		/// </summary>
+		/// <param name="age">
+		///		integer greater than 0 representing user's age
+		/// </param>
+		/// <param name="gender">
+		///		enum representing the user's gender (F, M, O)
+		/// </param>
+		public void SetUserData(int age = 0, Gender gender = Gender.None)
```

Comment:
Calling this method will overwrite the Segment array and delete any previously assigned segments which is likely to run into issues when a developer is implementing the SDK. 

The SDK should provide the ability to add or remove the age or gender individually without modifying other elements of the Segment array which can be done with individual methods for each piece of data. It would also be good from a product standpoint to provide syntactically familiar getters and setters by implementing properties for each field that wraps the underlying functionality.